packages/core/src/components/AvatarGroup/AvatarGroupCounter.tsx

```diff
@@ -73,13 +76,25 @@ const AvatarGroupCounter: React.FC<AvatarGroupCounterProps> = ({
     counterMaxDigits,
     counterPrefix,
     counterValue,
+    disabled,
     noAnimation
   ]);
 
   if (!counterTooltipAvatars.length && !counterValue) {
     return null;
   }
 
+  if (disabled) {
+    return (
+      <div
+        ref={counterContainerRef}
+        className={cx(styles.counterContainer, styles.disabled, counterSizeStyle, counterColorStyle)}
+      >
+        {counterComponent()}
+      </div>
+    );
+  }
```

Comment:
Why do we need both a wrapper and the inner counter to be disabled?

Reply:
Because this component uses a counter that already has a fixed background color, so it would look like this if we don't target the inner element as well 
<img width="318" alt="Screenshot 2024-05-27 at 11 30 18" src="https://github.com/mondaycom/vibe/assets/18269880/68db5297-1f2e-4538-a2b4-c6b466d08770">